src/container/MovieContainer.js

```diff
@@ -17,9 +16,9 @@ const MovieContainer = ({ movies, filterMovies, isLoaded }) => {
   })
 
   if (isLoaded) {
+    document.querySelector('.search-bar').classList.remove('hidden')
     return (
```

Comment:
Did some more research on this and I found a few articles suggesting that using querySelectors in React is not best practice. We can look at other potential solutions if time allows but I'm not too worried about it. My idea was to try and save search queries in state to pass around between the components, but this seems much simpler and uses much less code.